wormhole-connect/src/routes/sdkv2/route.ts

```diff
@@ -71,20 +74,28 @@ export class SDKv2Route {
     const fromChainSupported = supportedChains.includes(fromContext.chain);
     const toChainSupported = supportedChains.includes(toContext.chain);
 
-    const supportedDestinationTokens = await this.rc.supportedDestinationTokens(
-      sourceToken,
-      fromContext.context,
-      toContext.context,
-    );
-
-    const toTokenSupported = !!supportedDestinationTokens.find((tokenId) => {
-      return isSameToken(tokenId, destToken);
-    });
+    if (!fromChainSupported || !toChainSupported) {
+      return false;
+    }
 
-    const isSupported =
-      fromChainSupported && toChainSupported && toTokenSupported;
+    try {
+      const cacheKey = `isRouteSupported-${sourceToken.address}-${fromChain}-${toChain}`;
```

Comment:
Are there any stats about how many RPC calls caching saves us?

Reply:
on a huddle with Dave I witnessed Connect make zero calls when switching back to a chain that had been previously selected.

Reply:
With just SUI and ETH rpcs enabled, it brought calls down from 36 when opening up the view to select a destination down to 24, then it cached them so that when re-opening the same network tab there would be zero new calls.